src/legacy/blueprint-parser.coffee

```diff
@@ -74,8 +74,8 @@ getLocalAst = ({code, blueprintId, sourcemap, emitter}, cb) ->
     # Parsing metric
     t = process.hrtime()
 
-    protagonist.parse code, options, (err, result) ->
-
+    drafter = new Drafter(options)
```

Comment:
that's why I added YMMV :) only wanted to keep it consistent with the rest (otherwise I completely agree with your commit message)